src.ts/transaction.ts

```diff
@@ -27,6 +27,9 @@ export class AccessKey extends Assignable {
 }
 
 export function fullAccessKey(): AccessKey {
+    if (arguments.length != 0) {
+        throw new Error(`Wrong number of arguments ${arguments.length}: expected none`);
```

Comment:
Let's create specific error here https://github.com/nearprotocol/nearlib/blob/master/src.ts/utils/errors.ts

Reply:
I think it's also best to include arguments into error message instead of just length (faster to debug in most cases)